style.css

```diff
@@ -114,3 +102,92 @@ body {
 .nav-item {
   padding-left: 24px;
 }
+
+
+
+.right-col {
+  width: 350px;
+  display: flex;
+  flex-flow: column;
+}
+.search-box {
+  background-color: var(--items-bg-color);
+  border-radius: 25px;
+  height: 52px;
+  margin-top: 17px;
+}
+.trends-container {
+  background-color: var(--items-bg-color);
+  border-radius: 25px;
+  margin: 15px 0px 15px;
+}
+.box-header{
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 20px 0;
+  
+  
+}
+.trends-title{
+  margin: 0;
+}
+.settings{
+  padding: 8px;
+  color: var(--text-color);
+  border-radius: 50%;
+}
+.settings:hover{
+  background-color: var(--trends-bg-hover-items-color);
+}
+.trends{
+  list-style-type: none;
+  padding-left: 0;
+  margin: 0;
+}
+.trend{
+  padding: 10px;
+  padding-left: 20px;
+  cursor: pointer;
+}
+
+.trend:hover{
+  background-color: var(--trends-bg-hover-items-color);
+}
+.hashtag{
+  font-size: 14px;
+color: var(--trend-hashtag-tweets-color);
+}
+.trend-name{
+margin-top: 2px;
+margin-bottom: 2px;
+}
+.number-of-tweets{
+  font-size: 14px;
+  color: var(--trend-hashtag-tweets-color);
+  
+}
+.trend-more-info{
+  padding: 8px 10px;
+  color: var(--trend-hashtag-tweets-color);
+  border-radius: 50%;
+  float: right;
+  
+}
+.trend-more-info:hover{
+  background-color: var(--trend-more-info-bg-color);
+  color: var(--trend-more-info-color);
+  
+}
+.box-footer{
+  display: flex;
+}
+.show-more{
+  text-decoration: none;
+  color: var(--show-more-color);
+  border-radius: 0 0 25px 25px;
+  flex-grow:2;
```

Comment:
wouldn't `width: 100%;` make more sense here? this is not part of a flex box anyway